setup.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+
+from setuptools import setup, find_packages
+
+import os
+import shutil
+
+with open('README.md', 'r') as fp:
+    readme = fp.read()
+
+pkgs = find_packages('src', exclude=['data'])
+print('found these packages:', pkgs)
+
+src_dir = os.path.dirname(__file__)
+shutil.copytree(os.path.join(src_dir, 'ml'), os.path.join(src_dir, 'hdmf_ml', 'schema'), dirs_exist_ok=True)
```

Comment:
Is this necessary? Removing this makes the conversion to `pyproject.toml` easier.

Reply:
No... the schema just needs to be moved into the package directory so that it can be treated as package data.